Makefile

```diff
@@ -128,7 +128,20 @@ test-ember:
 	@echo "--> Running ember tests"
 	@cd ui && yarn run test-oss
 
-ember-ci-test:
+ember-ci-test: # Deprecated, to be removed soon.
```

Comment:
Not blocking: I think we could probably just remove this without a deprecation notice.

Reply:
I know this looks like noise and isn't referenced anywhere, but it's hard to know if any _people_ rely on this make target, so thought I'd give them a chance to find out the easy way rather than having to waste their time trying to work out what happened. It seems most PRs will be reviewed by only a small percentage of committers, so I don't think we can purely rely on that for communication. Git history would be enough though... Hmm, I prefer to play it safe and remove this soon rather than now.